alcs-api/src/application/application.entity.ts

```diff
@@ -31,13 +31,6 @@ export class Application extends Base {
   @Column()
   applicant: string;
 
-  @AutoMap()
-  @Column({
-    type: 'uuid',
-    nullable: true,
-  })
-  assigneeUuid: string;
```

Comment:
why did you decide to remove assigneUuid? I can see references to these fields, is the feature still working properly?

Reply:
I moved it down to be next to assignee instead of being near the top, it is still there